documentation/general/program-scale-testing-requirements-draft.md

```diff
@@ -228,19 +228,19 @@ integrators to track and test the designs in a common manner.
     saturated at 100Gbps for the duration of test runtime. At the same
     time we want as close to 100Gbps without losing packets.
 
--   Inactivity based aging timer of 1 second.
+-   Inactivity based aging timer of 5 second.
 
     -   All TCP connections should be deleted from the table after the
         test completes.
 
     -   The connection table should therefore be zero.
 
-    -   All UDP bi-directional flows need to age out before the 1 second
+    -   All UDP bi-directional flows need to age out before the 5 second
         interval to allow for new UDP bi-directional flows to be
         established. If everything works as advertised, we should never
         see the connection table go above the 8M connections. If we do
```

Comment:
Should "8M" be replaced with a different number?

Reply:
corrected to 32M flows or maybe it should be 15M only